internal/saucecloud/cucumber.go

```diff
@@ -56,7 +56,7 @@ func (r *CucumberRunner) RunProject() (int, error) {
 		return 1, err
 	}
 
-	fileURI, err := r.remoteArchiveProject(r.Project, r.Project.RootDir, r.Project.Sauce.Sauceignore, r.Project.DryRun)
+	filesURI, err := r.remoteArchiveProject(r.Project, r.Project.RootDir, r.Project.Sauce.Sauceignore, r.Project.DryRun)
```

Comment:
The correct pluralization would be:
```suggestion
	fileURIs, err := r.remoteArchiveProject(r.Project, r.Project.RootDir, r.Project.Sauce.Sauceignore, r.Project.DryRun)
```

`filesURI` would mean there's _one_ URI that refers to multiple files.